bin/studio-cli.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# The least terrible way to resolve a symlink to its real path.
+function realpath() {
+  /usr/bin/perl -e "use Cwd;print Cwd::abs_path(@ARGV[0])" "$0";
+}
+
+CONTENTS="$(command dirname "$(command dirname "$(command dirname "$(realpath "$0")")")")"
+BINARY_NAME="$(TERM=dumb command ls "$CONTENTS/MacOS/")"
+ELECTRON="$CONTENTS/MacOS/$BINARY_NAME"
+CLI="$CONTENTS/Resources/app/cli/index.js"
```

Comment:
Noting that I'm not sure yet what the path will be in the app output and if there are any hard blockers we need to take into account. This seems like a good start 👍